test/integration/build/test_build.py

```diff
@@ -7,7 +7,25 @@
 from utils import run_command_and_get_output
 from utils import get_rockspec_path
 
+from project import Project
+from project import INIT_NO_CARTRIDGE_FILEPATH
 from project import set_and_return_whoami_on_build
+from project import remove_all_dependencies, remove_project_file, replace_project_file
+
+
+@pytest.fixture(scope="function")
+def project_with_capital_letters_name(cartridge_cmd, short_tmpdir):
+    project = Project(cartridge_cmd, 'App-withoutDependencies01', short_tmpdir, 'cartridge')
+
+    remove_all_dependencies(project)
+
+    # This is necessary, because default app config has parameter `stateboard: true`
```

Comment:
Use something like "Remove file with Cartridge configuration" instead of "This".